doc/source/getting_started/faq.rst

```diff
@@ -80,5 +80,64 @@ data such as the mesh, you can use this code:
     sys.modules[__name__].__dict__.clear()
 
 
-However, a more efficient way is to clear MAPDL using the ``mapdl.clear()`` method. You
-can also exit and restart MAPDL.
+However, a more efficient way is to clear MAPDL using the 
+:meth:`clear() <ansys.mapdl.core.Mapdl.clear>`
+method. You can also exit and restart MAPDL.
+
+
+Why PyMAPDL results are different than the ones shown in the MAPDL GUI?
+=======================================================================
+
+Listing results
+---------------
+
+There might be several reasons why there is a difference between the results
+shown in the MAPDL GUI and the ones obtained using PyMAPDL. The most common
+reason is that the MAPDL GUI is using a different configuration than the one
+used by PyMAPDL.
+
+In the MAPDL GUI the graphics configuration can change how the results are shown.
+By default, the graphics configuration is set to ``Power graphics``.
+However, PyMAPDL connects to an MAPDL instance running in batch mode which by default
+uses ``Full Graphics`` configuration. This can affect the averaging as well.
```

Comment:
```suggestion
In the MAPDL GUI, the graphics configuration can change how the results are shown.
By default, the graphics configuration is set to ``Power graphics``.
However, PyMAPDL connects to an MAPDL instance running in batch mode, which by default
uses the ``Full Graphics`` configuration. This difference in the graphics configuration can
affect averaging as well.
```
Should the graphics in the option setting be lower case for "Power graphics" but upper case for "Full graphics"? It seems like one is a typo.

Reply:
Fixed!